### DISCUSSION TITLE
I need help with a conundrum

### DISCUSSION THREAD

**Discussion by Ilovetoocode** [BepInEx/BepInEx#538] [Help and Q&A] [answered]
Basically, I wanna use this on Linux, but I'm trying to patch a game that's a Windows exe that I'm running in Proton, meaning i don't think i can use the Linux patcher because that's built for Linux and the game runs on Windows. What's the solution on this?

**Comment by Windows10CE** [comment, marked as answer]
https://docs.bepinex.dev/articles/advanced/proton_wine.html

**Reply by Ilovetoocode** [reply]
Thanks! Worked perfectly!
